src/components/Contexts/StepperDialogProvider.jsx

```diff
@@ -1,27 +1,39 @@
-import React, { createContext, useCallback, useState } from 'react'
+import React, { createContext, useCallback, useEffect, useState } from 'react'
 
 const StepperDialogContext = createContext()
 
 const StepperDialogProvider = ({ children }) => {
   const [isStepperDialogOpen, setIsStepperDialogOpen] = useState(false)
   const [stepperDialogTitle, setStepperDialogTitle] = useState('')
   const [allCurrentPages, setAllCurrentPages] = useState([])
-  const [currentPage, setCurrentPage] = useState(1)
+  const [allCurrentPagesSafed, setAllCurrentPagesSafed] = useState([])
```

Comment:
Not sure to understand what "safed" means in this context

Reply:
En effet, c'est un oubli de renommage d'une première verison..
Que dirais-tu du `allCurrentPagesWithOccurrences` ? :)

Reply:
Plus clair déjà.

Autre proposition : 
Le `setAllCurrentPages` utilisé par le `Placeholder` est en fait plutôt un `setAllCurrentPagesModels` ou encore `setAllCurrentPagesDefinitions` (pour être raccord avec le nom du json) car on manipule le modèle déclaré dans le json.
Ainsi ça laisse le nom `allCurrentPages` disponible pour la version "withOccurences" (mais qui n'a pas besoin de ce préfixe du coup) qui correspond à la version interprétée de la définition.